src/test/cypress/integration/ProgrammingExercise.spec.ts

```diff
@@ -55,13 +48,13 @@ describe('Programming Exercise Management', () => {
         it('Creates a new programming exercise', function () {
             cy.login(admin, '/');
             navigationBar.openCourseManagement();
-            courseManagementPage.openExercisesOfCourse(courseName, courseShortName);
+            courseManagementPage.openExercisesOfCourse(course.title, course.shortName);
             cy.get('#jh-create-entity').click();
             cy.url().should('include', '/programming-exercises/new');
             cy.log('Filling out programming exercise info...');
             programmingCreation.setTitle(programmingExerciseName);
             programmingCreation.setShortName(programmingExerciseShortName);
-            programmingCreation.setPackageName(packageName);
+            programmingCreation.setPackageName('de.test');
```

Comment:
would it not make sense to include this as a default in the programmingCreation pageobject as well?

Reply:
I don't think it makes sense in this case. When you call a method on a pageobject, where it types something in, you should always have to provide a parameter. In this case I think it is more misleading to have a defaul value because then you can call a setter method without actually providing a new value.
Also this ticket was about adding default values for requests. If I start providing default values for pageobjects I should probably do it in another PR since there are a lot of pageobjects.